code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -40,7 +40,7 @@
 	if (is_drowsy())
 		tally += 6
 
-	if (!(species.flags & IS_MECHANICAL))	// Machines don't move slower when cold.
+	if (!(species.flags & NO_COLD_SLOWDOWN))	// Bugs and machines don't move slower when cold.
```

Comment:
you never added this flag to IPCs.

Reply:
> you never added this flag to IPCs.

![image](https://user-images.githubusercontent.com/36427443/222981160-ab4a2fc4-8961-46a2-80fd-ff2b8211fda0.png)
so what does this do?

Reply:
my bad. was expecting it to be added to the IPC DM files.